deployer/hub.py

```diff
@@ -400,8 +400,8 @@ def get_generated_config(self, auth_provider: KeyProvider, secret_key):
                                 dedent(
                                     f"""\
                                     while true; do git fetch origin;
-                                    if [[ $(git ls-remote --heads origin {self.spec["name"]} | wc -c) -ne 0 ]]; then
-                                        git reset --hard origin/{self.spec["name"]};
+                                    if [[ $(git ls-remote --heads origin {self.cluster}-{self.spec["name"]} | wc -c) -ne 0 ]]; then
```

Comment:
I might be wrong here, but should this be `self.cluster.spec["name"]`?

Reply:
You're absolutely right! Thanks @sgibson91! I just added a commit to address it.